packages/bpk-component-popover/src/BpkPopover-test.tsx

```diff
@@ -16,99 +16,139 @@
  * limitations under the License.
  */
 
-/* @flow strict */
-
 import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 
 import BpkPopover from './BpkPopover';
 
 describe('BpkPopover', () => {
+  let onCloseSpy = jest.fn();
+
+  beforeEach(() => {
+    onCloseSpy = jest.fn();
+  })
+
   it('should render correctly', () => {
-    const { asFragment } = render(
+    const target = (<button type="button">My target</button>);
+    render(
       <BpkPopover
         id="my-popover"
         onClose={() => null}
         label="My popover"
         closeButtonText="Close"
+        target={target}
+        isOpen
       >
         My popover content
       </BpkPopover>,
     );
-    expect(asFragment()).toMatchSnapshot();
+
+    expect(screen.getByText('My popover content')).toBeVisible();
   });
 
-  it('should render correctly with "closeButtonProps" provided', () => {
-    const { asFragment } = render(
+  it('should render without an arrow', () => {
+    const target = (<button type="button">My target</button>);
+    render(
       <BpkPopover
         id="my-popover"
         onClose={() => null}
         label="My popover"
         closeButtonText="Close"
-        tabIndex="0"
-        closeButtonProps={{ tabIndex: 0 }}
+        target={target}
+        isOpen
+        showArrow={false}
       >
         My popover content
       </BpkPopover>,
     );
-    expect(asFragment()).toMatchSnapshot();
+    expect(screen.getByText('My popover content')).toBeVisible();
   });
 
-  it('should render correctly with "padded" attribute equal to false', () => {
-    const { asFragment } = render(
+  // it('should render to the left', () => {
```

Comment:
Should we delete this or uncomment it?

Reply:
Ah yeah this is one I need to action to make it work as it would be good to test it but it was giving weird results!